src/components/common/AsideNavbar/AsideNavbar.tsx

```diff
@@ -36,45 +38,63 @@ function AsideNavbar() {
+                            <BiHouseHeart height="1.5em" width="1.5em" />Home
                         </NavLink>
                     </li>
                     <li>
                         <NavLink style={getActiveStyle}
                             to={ROUTE_PATH_ArchivePage}>
-                            <BiArchive  height="1.7em" width="1.7em"/>
+                            <BiArchive height="1.5em" width="1.5em" />
                             Archive
                         </NavLink>
                     </li>
+                    <li>
+                        <NavLink style={getActiveStyle}
+                            to={ROUTE_PATH_LabelPage}>
+                            <BiTags height="1.5em" width="1.5em" />
+                            Label
+                        </NavLink>
+                    </li>
                     <li>
                         <NavLink style={getActiveStyle}
                             to={ROUTE_PATH_TrashPage}>
-                            <BiTrash  height="1.7em" width="1.7em"/>
+                            <BiTrash height="1.5em" width="1.5em" />
                             Trash
                         </NavLink>
                     </li>
-                    <li onClick={
-                        () => {
-                            console.log("login");
-                            setmodalToggle(true);
-                        }
-                    }>
-                        <BiPersonHearts  height="1.7em" width="1.7em"/>Profile
-                    </li>
+                    {
+                        !auth.user ?
+
+                            <span onClick={
+                                () => {
+                                    console.log("login");
+                                    setmodalToggle(true);
+                                }} className='create-btn' style={{ width: "100%" }}>Login<BiBoxArrowInLeft height="1.5em" width="1.5em" /></span>
+
+                            : <li onClick={
+                                () => {
+                                }
+                            }>
+                                <BiPersonHearts height="1.5em" width="1.5em" /> Profile
+                            </li>
+                    }
                 </ul>
             </div>
             <div className='aside-nav-logout'>
-                { auth.user ? "@" + auth.user?.firstName + auth.user?.lastName : ""}
-                <span className='logout-btn'
-                    onClick={
-                        () => {
-                            auth.logoutUser(() => {
-                                navigate("/");
-                            })
-                        }
-                }>
-                    <BiBoxArrowRight/>
-                </span>
+                {
+                    auth.user ?
+                        <> {"@" + auth.userState?.firstName + auth.userState?.lastName}
```

Comment:
Instead of doing it in jsx. You can make a custom variable and do it there.

Reply:
ok